sos/report/plugins/spyre.py

```diff
@@ -0,0 +1,74 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+#
+# This plugin enables collection of logs for system with IBM Spyre card
+
+import re
+import pyudev
```

Comment:
You can't rely on presence of this library, and default CentOS Stream 9 (or e.g. Fedora 42) installation does not have it.